modules/govuk_jenkins/templates/jobs/prune_elasticsearch_snapshots.yaml.erb

```diff
@@ -0,0 +1,34 @@
+---
+- job:
+    name: prune-elasticsearch-snapshots
+    display-name: prune-elasticsearch-snapshots
+    project-type: freestyle
+    description: "<p>Delete old Elasticsearch snapshots</p>
+      <p>This is run to delete old (7 days) Elasticsearch snapshots.</p>"
+    builders:
+        - shell: |
+            ssh deploy@search-1.api '(cd /var/apps/rummager; RUMMAGER_INDEX=all govuk_setenv rummager bundle exec rake rummager:snapshot:clean)'
+    triggers:
+        - timed: '5 4 * * *' # to update
```

Comment:
Possibly, although early morning seems like a reasonable time to run this. What do you think?